docs/index.md

```diff
@@ -46,7 +46,7 @@ The image below shows the timing around how the Mobi Planner turns Bookings and
 
 ## Regular Planning Overview
 
-1. When a Booking comes in via the AWS Kinesis Data Stream, it gets ingested but not planned until the **Planning Window** begins for the relevant Destination and date. For most Destinations, the Planning Window begins 7 days before the date of travel and ends 24 hours before the time of travel.
+1. When a Booking comes in via the AWS Kinesis Data Stream, it gets ingested but not planned until the **Planning Window** (What is the planning window definition? what value do we set it now? 14 days? 7 days according to the image? Maybe we can rename this to **regular planning window**?) begins for the relevant Destination and date. For most Destinations, the Planning Window begins 7 days before the date of travel and ends 24 hours before the time of travel.
```

Comment:
Overall I think this looks great!!! After your organization my parts look a looooooot better! Many many thank you :)
I only have some questions regarding planning window.